pkg/channel/consolidated/reconciler/controller/kafkachannel.go

```diff
@@ -616,6 +618,8 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, kc *v1beta1.KafkaChannel)
 			return err
 		}
 	}
-	r.consumerGroupWatcher.Forget(string(kc.ObjectMeta.UID))
+	if r.consumerGroupWatcher != nil {
+		r.consumerGroupWatcher.Forget(string(kc.ObjectMeta.UID))
+	}
```

Comment:
wat is that?

Reply:
`consumerGroupWatcher` becomes nil when the bootstrap server is unreachable. Like:
1. Have `config-kafka` has proper bootstrap server
2. Create a Kafka channel
3. Change bootstrap server in `config-kafka` to nonsense
4. `consumerGroupWatcher` becomes nil
5. Couldn't delete Kafka channel afterwards --> panic

This change adds a null check that's all

Reply:
@aliok let's capture that in the comments?  Please ? :smile: